src/service/programService.js

```diff
@@ -171,6 +170,198 @@ function updateProgram(req, response) {
   });
 }
 
+function publishProgram(req, response) {
+  var data = req.body;
+  var rspObj = req.rspObj;
+  if (!data.request || !data.request.program_id || !data.request.channel) {
```

Comment:
why are looking for channel in the request? @ahghatol

Reply:
@twsvaishali 
We need this for collection copy